src/components/interactive-builder/modals/question/form-field-context.tsx

```diff
@@ -11,6 +16,8 @@ interface FormFieldContextType {
   setIsConceptValid?: React.Dispatch<React.SetStateAction<boolean>>;
   updateParentFormField?: (updatedFormField: FormField) => void;
   isObsGrouped?: boolean;
+  additionalAnswers: AdditionalAnswer[];
+  setAdditionalAnswers: React.Dispatch<React.SetStateAction<AdditionalAnswer[]>>;
```

Comment:
The additional answers are only used within the `select-answers` component, so adding them in to the context is unnecessary. All current context variables exist specifically because they need to be shared across multiple components. Adding answers to the context seems like an excessive solution just to support answer reordering functionality, which is strictly a `select-answers `component feature.